config/i18n/it/common.php

```
@@ -3539,8 +3552,12 @@
     'ui.settings.echomail_digest_none' => 'Off',
     'ui.settings.echomail_digest_daily' => 'Giornaliero',
     'ui.settings.echomail_digest_weekly' => 'Settimanale',
-    'ui.settings.echomail_digest_help' => 'Ricevi un’email periodica che riassume i nuovi messaggi nelle aree echo sottoscritte. Richiede un indirizzo email valido e SMTP configurato.',
-    'ui.settings.onboarding.title' => 'Onboarding echomail',
+    ‘ui.settings.echomail_digest_help’ => ‘Ricevi un’email periodica che riassume i nuovi messaggi nelle aree echo sottoscritte. Richiede un indirizzo email valido e SMTP configurato.’,
```

PHPStan (2.1.51)

[error] Cannot use empty array elements in arrays on line 3555


(phpstan.parse)